### PR TITLE
Norwegian translation january2018

### PR DESCRIPTION
**Pull request by ahaarrestad**
I have updated the norwegian translation to be a bit more norwegian and quite consistent. 

Can you take a look, @dabear  ?

The branch built fine in my fork, @jamorham

### CODE REVIEW

File: app/src/main/res/values-nb/strings-nb.xml [hunk 1]
```diff
@@ -1,501 +1,830 @@
...
+    <string name="pref_title_mongodb_collection">Samlingsnavn</string>
+    <string name="pref_dialog_mongodb_collection">Skriv inn navnet på samlingen</string>
+    <string name="pref_message_mongodb_collection">Dette er navnet på samlingen hvor CGM data vil bli lagret</string>
+
+    <string name="pref_title_mongodb_device_status_collection">Samlingsnavn for enhetsstatus</string>
+    <string name="pref_dialog_mongodb_device_status_collection">Skriv inn samlingsnavnet for Enhetsstatus"</string>
+    <string name="pref_message_mongodb_device_status_collection">Dette er navnet på samlingen der batteriet og andre enhetsdata vil bli lagret</string>
+
+    <!-- API Settings -->
+    <string name="pref_title_api">Nightscout synkronisering (REST)</string>
```
Comment (dabear) [inline app/src/main/res/values-nb/strings-nb.xml]: Nightscout-synkronisering (REST)

File: app/src/main/res/values-nb/strings-nb.xml [hunk 2]
```diff
@@ -1,501 +1,830 @@
...
+    <string name="pref_title_api_enabled">Aktivert</string>
+    <string name="pref_summary_api_enabled">REST API er et alternativ til direkte mongodb opplasting</string>
```
Comment (dabear) [inline app/src/main/res/values-nb/strings-nb.xml]: mongodb-opplasting

File: app/src/main/res/values-nb/strings-nb.xml [hunk 3]
```diff
@@ -1,501 +1,830 @@
...
+    <string name="pref_title_api_url">Grunn-URL</string>
```
Comment (dabear) [inline app/src/main/res/values-nb/strings-nb.xml]: Base-URL would be the correct form here. "Grunn" seems incorrect

File: app/src/main/res/values-nb/strings-nb.xml [hunk 4]
```diff
@@ -1,501 +1,830 @@
...
+    <string name="pref_dialog_api_url">Skriv inn grunn-URL for API</string>
```
Comment (dabear) [inline app/src/main/res/values-nb/strings-nb.xml]: Same here. Base-URL

File: app/src/main/res/values-nb/strings-nb.xml [hunk 5]
```diff
@@ -1,501 +1,830 @@
...
+    <string name="pref_message_api_url">Dette er bare grunn-URL, opplaster vil automatisk legge til /oppføringer for POST. API_SECRET på serveren skal samsvare med yourpassphrase i denne innstillingen.</string>
```
Comment (dabear) [inline app/src/main/res/values-nb/strings-nb.xml]: Base-URL

File: app/src/main/res/values-nb/strings-nb.xml [hunk 6]
```diff
@@ -1,501 +1,830 @@
...
+    <string name="pref_default_api_url">https://yourpassphrase@{YOUR-SITE}.azurewebsites.net/api/v1/</string>
+
+    <!-- wifi Settings -->
+    <string name="wifi_recievers_dialog_message">Kommaseparert liste av ip:port (for eksempel 37.142.132.220:50005,37.142.132.220:50010,mongodb://user:pass@ds053958.mongolab.com:53958/db/collection)</string>
+    <string name="wifi_recievers_dialog_title">Skriv inn IP-adresser og porter mottakere (inkludert mongodb adresser hvis nødvendig)</string>
+
+    <!-- Location Settings -->
+    <string name="location_not_found_title">Plassering er ikke aktivert</string>
```
Comment (dabear) [inline app/src/main/res/values-nb/strings-nb.xml]: Stedstjenester

File: app/src/main/res/values-nb/strings-nb.xml [hunk 7]
```diff
@@ -1,501 +1,830 @@
...
+    <string name="location_not_found_message">For at blåtannoppdagelse skal virke med nyere enheter, må plassering være aktivert. xDrip kan ikke spore posisjonen din, og det kan deaktiveres etter at paringen er vellykket.</string>
```
Comment (dabear) [inline app/src/main/res/values-nb/strings-nb.xml]: Stedstjenester

File: app/src/main/res/values-nb/strings-nb.xml [hunk 8]
```diff
@@ -1,501 +1,830 @@
...
+    <string name="pref_pebble_display_trend">Vis Trend</string>
+    <string name="pref_summary_display_trend">Vis BG trenden på xDrip Pebble urskiven.</string>
+    <string name="pref_pebble_display_high_line">Vis Høy Linje</string>
+    <string name="pref_pebble_display_low_line">Vis Lav Linje</string>
+
+
+    <string name="toast_crash">Noe gikk galt :( en rapport har blitt sendt for å hjelpe til å løse problemet.</string>
+
+    <string name="taskerConfigInstruction">Bare klikk på Tick-ikonet her for å bekrefte. Du bør ikke trenger å endre konfigurasjonen.</string>
+    <string name="taskerConfiguration">Tasker konfigurasjon</string>
```
Comment (dabear) [inline app/src/main/res/values-nb/strings-nb.xml]: Tasker-konfigurasjon

File: app/src/main/res/values-nb/strings-nb.xml [hunk 9]
```diff
@@ -1,501 +1,830 @@
...
+    <string name="title_activity_display_preferences">Vis Innstillinger</string>
+
+    <string name="export_submenu">Import / Eksport funksjoner</string>
+
+    <string name="action_resend_to_watch">Sende glukose til klokke</string>
+    <string name="action_open_watch_setting">Åpne Innstillinger på Wear</string>
+    <string name="action_sync_watch_db">Tilbakestill Wear DB</string>
+
+    <string name="erase">Fjern</string>
+    <string name="interface_tips_from_start">Alle grensesnitt tips vil vise igjen fra begynnelsen</string>
```
Comment (dabear) [inline app/src/main/res/values-nb/strings-nb.xml]: grensesnitt-tips

File: app/src/main/res/values-nb/strings-nb.xml [hunk 10]
```diff
@@ -1,501 +1,830 @@
...
+    <string name="showcase_note_long">Notatknappen ber nå om taleregistrering.\n\nHold inne lenge for å legg til med tastatur eller endre inntastingsvalg</string>
+    <string name="redo_button">Gjør På Ny Knapp</string>
+    <string name="showcase_redo">Gjør På Ny Knappen kan gå fremover gjennom ugjort Behandlingssimuleringer, Kalibreringer og merknader\n\nEtter 30 minutter vil den skjules igjen</string>
+    <string name="undo_button">Angreknapp</string>
+    <string name="showcase_undo">Angreknappen kan gå bakover gjennom behandlingssimuleringer, Kalibrering og Notater\n\nEtter 30 minutter vil den skjules igjen</string>
+    <string name="add_note">Legge til notat</string>
+    <string name="blood_test">Blodprøve</string>
+    <string name="share">Del</string>
+    <string name="title_activity_profile_editor">Rediger Behandlingprofil</string>
+    <string name="first_use_menu_to_scan">Bruk først menyen til å søke etter din BT-enheten eller endre datakilde i Innstillinger!</string>
```
Comment (dabear) [inline app/src/main/res/values-nb/strings-nb.xml]: "BT-enhet", not "BT-enheten"

File: app/src/main/res/values-nb/strings-nb.xml [hunk 11]
```diff
@@ -1,501 +1,830 @@
...
+    <string name="first_configure_ip_address">Først konfigurere wifi-wixel-leserens ip-adresser eller parakeet URL</string>
+    <string name="now_start_your_sensor">Starte sensoren nå, eller sjekk innstillinger</string>
+    <string name="please_wait_while_sensor_warms_up">Vennligst vent mens sensoren varmer opp! (</string>
+    <string name="minutes_with_bracket">" minutter)"</string>
+    <string name="possible_bad_calibration">Mulig dårlig kalibreriskurve, vennligst drikk et glass vann og vask hendene, og rekalibrer om noen litt!</string>
+    <string name="please_wait_need_two_readings_first">Vennligst vent, trenger 2 avlesninger fra senderen først.</string>
+    <string name="please_enter_two_calibrations_to_get_started">Legg inn to kalibreringer for å komme i gang!</string>
+    <string name="now_pair_with_your_dexcom_share">Par med Dexcom Share</string>
```
Comment (dabear) [inline app/src/main/res/values-nb/strings-nb.xml]: Parr? Two "r"'s

File: app/src/main/res/values-nb/strings-nb.xml [hunk 12]
```diff
@@ -1,501 +1,830 @@
...
+    <string name="now_choose_start_sensor_in_settings">Velger start sensoren i innstillingene</string>
```
Comment (dabear) [inline app/src/main/res/values-nb/strings-nb.xml]: "Velg"

File: app/src/main/res/values-nb/strings-nb.xml [hunk 13]
```diff
@@ -1,501 +1,830 @@
...
+    <string name="pref_summary_force_wearG5">Tving telefonen til å bruke Wear innsamlingstjenesten</string>
+    <string name="pref_disable_wearG5_on_lowbattery">Deaktiver tvang ved lavt batteri</string>
+    <string name="pref_summary_disable_wearG5_on_lowbattery">Deaktiver tvang av Wear innsamlingstjeneste ved Lavt batterivarser fra Wear</string>
+    <string name="pref_disable_wearG5_on_missedreadings">Deaktiver tvang ved tapte avlesninger</string>
+    <string name="pref_summary_disable_wearG5_on_missedreadings">Deaktiver Tvang av Wear innsamlingstjenste ved Minutter siden forrige avlesning fra Wear</string>
+    <string name="pref_disable_wearG5_on_missedreadings_level">Minutter siden forrige avlesning fra Wear</string>
+    <string name="pref_node_wearG5">Enhet som kjører innsamlingstjenesten</string>
+    <string name="pref_summary_node_wearG5">Tving telefonen til å bruke denne Wear innsamlingstjenesten</string>
+
+    <string name="pref_sync_wear_logs">Synkroniser Wear Logger</string>
```
Comment (dabear) [inline app/src/main/res/values-nb/strings-nb.xml]: Wear-logger

File: app/src/main/res/values-nb/strings-nb.xml [hunk 14]
```diff
@@ -1,501 +1,830 @@
...
+    <string name="message_to_display_when_bgl_hits">Melding som skal vises når BGL treffer spesiell verdi over</string>
+    <string name="text_to_display_when_hitting_special_value">Tekst som skal vises når du treffer spesiell verdi</string>
+    <string name="special_value">Spesiell verdi</string>
+    <string name="history">Historikk</string>
+    <string name="statistics">Statistikk</string>
+    <string name="snooze_alert">Utsett Alert</string>
+    <string name="less_common_settings">Mindre vanlige innstillinger</string>
+    <string name="view_recent_errors_warnings">Vis nylige Feil / Advarsler</string>
+    <string name="other_settings">Andre innstillinger</string>
+    <string name="older_gratuitous_wakelocks">Eldre umotiverte wakelocks som kan tappe batteriet ekstra, men kan være nødvendig for Nightwatch eller Android Wear</string>
```
Comment (dabear) [inline app/src/main/res/values-nb/strings-nb.xml]: "Umotiverte" seems incorrect, but I am uncertain of the correct translation here

File: app/src/main/res/values-nb/strings-nb.xml [hunk 15]
```diff
@@ -1,501 +1,830 @@
...
+    <string name="use_excessive_wakelocks">Bruk Overdreven Wakelocks</string>
+    <string name="speak_readings">Snakk inn Innlesninger</string>
+    <string name="extra_status_line">Ekstra Statuslinje</string>
+    <string name="dexcom_share_server_upload">Opplasting til Dexcom Share Server</string>
+    <string name="upload_data_to_dex_servers">Laste opp data til Dexcoms servere slik at du kan bruke dine data med Dexcom sine apper</string>
+    <string name="manage_followers">Administrer følgere</string>
+    <string name="manage_existing_followers">Administrer dine eksisterende følgere og invitere nye.</string>
+    <string name="glucose_calibration_and_other_alerts">Glukose, Kalibrering og andre varsler</string>
+    <string name="alert_volume_profile">Alarm Volume Profil</string>
```
Comment (dabear) [inline app/src/main/res/values-nb/strings-nb.xml]: alarmvolum

File: app/src/main/res/values-nb/strings-nb.xml [hunk 16]
```diff
@@ -1,501 +1,830 @@
...
+    <string name="done">Ferdig</string>
+    <string name="entering_a_calibration_now_will_override_your_previously_entered_calibration">Legges det inn en kalibrering nå, vil det overstyre tidligere kalibrering.</string>
+    <string name="enter_blood_glucose_value">Skriv inn blodsukkerverdi</string>
+    <string name="this_does_not_get_used_for_anything_its_just_to_keep_track_of_differences_for_algorithem_improvement_purposes">Dette blir ikke brukt til noe, det er bare å holde styr på forskjellen mellom algoritmene for forbedringsformål</string>
+    <string name="no_active_alert_exists">Ingen aktiv varsling finnes</string>
+    <string name="active_alert_exists_named">Aktiv varsling eksisterer med navn</string>
+    <string name="bracket_not_snoozed">(Ikke utsatt)</string>
+    <string name="all_alerts_disabled_until">"Alle varsler deaktivert inntil "</string>
+    <string name="low_alerts_disabled_until">"Lave varsler deaktivert inntil "</string>
+    <string name="high_alerts_disabled_until"
```
Comment (dabear) [inline app/src/main/res/values-nb/strings-nb.xml]: Vis Generelle- og innsamlingsinnstillinger

Comment (dabear) [inline app/src/main/res/values-nb/strings-nb.xml]: "Kun xDrip Plus sikkerhetsnøkkel-innstillinger" OR use the english term "security key"

Comment (dabear) [inline app/src/main/res/values-nb/strings-nb.xml]: Tast inn første blodverdi

Comment (dabear) [inline app/src/main/res/values-nb/strings-nb.xml]: Tast inn andre blodverdi

Comment (dabear) [inline app/src/main/res/values-nb/strings-nb.xml]: glukosedata

Comment (dabear) [inline app/src/main/res/values-nb/strings-nb.xml]: "Vis 8-timers gjennomsnittslinje" (drop the trailing "n")

Comment (dabear) [inline app/src/main/res/values-nb/strings-nb.xml]: Same here, drop the trailing "n"

Comment (dabear) [inline app/src/main/res/values-nb/strings-nb.xml]: Innstillinger for Prediktiv-model?

Comment (dabear) [inline app/src/main/res/values-nb/strings-nb.xml]: Glukosetall

Comment (dabear) [inline app/src/main/res/values-nb/strings-nb.xml]: RFDUINO-støtte

Comment (dabear) [inline app/src/main/res/values-nb/strings-nb.xml]: feillogger?

Comment (dabear) [inline app/src/main/res/values-nb/strings-nb.xml]: A1c-estimat

Comment (dabear) [inline app/src/main/res/values-nb/strings-nb.xml]: Bruk glukose fra plugin

Comment (dabear) [inline app/src/main/res/values-nb/strings-nb.xml]: tillegg->plugin

Comment (dabear) [inline app/src/main/res/values-nb/strings-nb.xml]: tillegg->plugin

Comment (dabear) [inline app/src/main/res/values-nb/strings-nb.xml]: tillegg->plugin.
 "xDrip algoritme"-> "xDrip-algoritme"

Comment (dabear) [inline app/src/main/res/values-nb/strings-nb.xml]: for klassisk xDrip-kalibreringsalgoritme (pre-2017).

"mer sannsynlig" (two "n"'s)

Comment (dabear) [inline app/src/main/res/values-nb/strings-nb.xml]: NFC-basert

Comment (dabear) [inline app/src/main/res/values-nb/strings-nb.xml]: "Noen enheter virker bedre når de**t** blir" -> "Noen enheter virker bedre når de blir"

Comment (dabear) [inline app/src/main/res/values-nb/strings-nb.xml]: Androidenheter

Comment (dabear) [inline app/src/main/res/values-nb/strings-nb.xml]: Datasynkronisering

Comment (dabear) [inline app/src/main/res/values-nb/strings-nb.xml]: Opplasting til Skyen

Comment (dabear) [inline app/src/main/res/values-nb/strings-nb.xml]: addresses->addresse**R**

Comment (dabear) [inline app/src/main/res/values-nb/strings-nb.xml]: LAN-opplasting

Comment (dabear) [inline app/src/main/res/values-nb/strings-nb.xml]: Dexcom-konto

Comment (dabear) [inline app/src/main/res/values-nb/strings-nb.xml]: Unsure if flair can be translated by "humør" here. I think flair means "style" in this context, thus the translation should be "stilfarger"

Comment (dabear) [reply app/src/main/res/values-nb/strings-nb.xml]: Also see the strings below, these would also have to be corrected

File: app/src/main/res/values-nb/strings-nb.xml [hunk 17]
```diff
@@ -1,501 +1,830 @@
...
+    <string name="title_activity_calibration_check_in">Kalibreringsaktivitet</string>
+    <string name="title_activity_usb_connected">USB-tilkoblet aktivitet</string>
+    <string name="snooze">Utsett</string>
+    <string name="snooze_hint">Utsett varsling i x minutter</string>
+    <string name="ok">OK</string>
+    <string name="cancel">Avbryt</string>
+    <string name="no">Nei</string>
+    <string name="pref_sensor_location_dont_ask_again"> Ikke spør meg igjen</string>
+
+    <string name="title_activity_error">"Feil- og Handlingslist"</string>
```
[the remark this answers is not in the document]
Comment (ahaarrestad) [reply app/src/main/res/values-nb/strings-nb.xml]: Done

File: app/src/main/res/values-nb/strings-nb.xml (hunk 1 above)
Comment (ahaarrestad) [reply app/src/main/res/values-nb/strings-nb.xml]: Done

File: app/src/main/res/values-nb/strings-nb.xml (hunk 2 above)
Comment (ahaarrestad) [reply app/src/main/res/values-nb/strings-nb.xml]: Done

File: app/src/main/res/values-nb/strings-nb.xml (hunk 3 above)
Comment (ahaarrestad) [reply app/src/main/res/values-nb/strings-nb.xml]: Done

File: app/src/main/res/values-nb/strings-nb.xml (hunk 4 above)
Comment (ahaarrestad) [reply app/src/main/res/values-nb/strings-nb.xml]: Done

File: app/src/main/res/values-nb/strings-nb.xml (hunk 5 above)
Comment (ahaarrestad) [reply app/src/main/res/values-nb/strings-nb.xml]: Done

File: app/src/main/res/values-nb/strings-nb.xml (hunk 6 above)
Comment (ahaarrestad) [reply app/src/main/res/values-nb/strings-nb.xml]: Done

File: app/src/main/res/values-nb/strings-nb.xml (hunk 7 above)
Comment (ahaarrestad) [reply app/src/main/res/values-nb/strings-nb.xml]: Done

File: app/src/main/res/values-nb/strings-nb.xml (hunk 8 above)
Comment (ahaarrestad) [reply app/src/main/res/values-nb/strings-nb.xml]: Done

File: app/src/main/res/values-nb/strings-nb.xml (hunk 9 above)
Comment (ahaarrestad) [reply app/src/main/res/values-nb/strings-nb.xml]: Done

File: app/src/main/res/values-nb/strings-nb.xml (hunk 10 above)
Comment (ahaarrestad) [reply app/src/main/res/values-nb/strings-nb.xml]: Done

File: app/src/main/res/values-nb/strings-nb.xml (hunk 11 above)
Comment (ahaarrestad) [reply app/src/main/res/values-nb/strings-nb.xml]: Done

File: app/src/main/res/values-nb/strings-nb.xml (hunk 12 above)
Comment (ahaarrestad) [reply app/src/main/res/values-nb/strings-nb.xml]: Done

File: app/src/main/res/values-nb/strings-nb.xml (hunk 13 above)
Comment (ahaarrestad) [reply app/src/main/res/values-nb/strings-nb.xml]: Done

File: app/src/main/res/values-nb/strings-nb.xml (hunk 14 above)
Comment (ahaarrestad) [reply app/src/main/res/values-nb/strings-nb.xml]: Trying with "unødvendige". google explains gratuitous as "done without good reason; uncalled for."

File: app/src/main/res/values-nb/strings-nb.xml (hunk 15 above)
Comment (ahaarrestad) [reply app/src/main/res/values-nb/strings-nb.xml]: Done

File: app/src/main/res/values-nb/strings-nb.xml (hunk 16 above)
Comment (ahaarrestad) [reply app/src/main/res/values-nb/strings-nb.xml]: Done

Comment (ahaarrestad) [reply app/src/main/res/values-nb/strings-nb.xml]: This was line 339. Done

Comment (ahaarrestad) [reply app/src/main/res/values-nb/strings-nb.xml]: Line 343. Done

Comment (ahaarrestad) [reply app/src/main/res/values-nb/strings-nb.xml]: Line 344. Done

Comment (ahaarrestad) [reply app/src/main/res/values-nb/strings-nb.xml]: line 422. Done

Comment (ahaarrestad) [reply app/src/main/res/values-nb/strings-nb.xml]: Line 523. Done

Comment (ahaarrestad) [reply app/src/main/res/values-nb/strings-nb.xml]: Line 525. Done

Comment (ahaarrestad) [reply app/src/main/res/values-nb/strings-nb.xml]: Which line is this?

Comment (ahaarrestad) [reply app/src/main/res/values-nb/strings-nb.xml]: Line 528. Done

Comment (ahaarrestad) [reply app/src/main/res/values-nb/strings-nb.xml]: Line 597. Done

Comment (ahaarrestad) [reply app/src/main/res/values-nb/strings-nb.xml]: Line 610. Done

Comment (ahaarrestad) [reply app/src/main/res/values-nb/strings-nb.xml]: Line 623. Done

Comment (ahaarrestad) [reply app/src/main/res/values-nb/strings-nb.xml]: Done

Comment (ahaarrestad) [reply app/src/main/res/values-nb/strings-nb.xml]: Done

Comment (ahaarrestad) [reply app/src/main/res/values-nb/strings-nb.xml]: Done

Comment (ahaarrestad) [reply app/src/main/res/values-nb/strings-nb.xml]: Done

Comment (ahaarrestad) [reply app/src/main/res/values-nb/strings-nb.xml]: Done

Comment (ahaarrestad) [reply app/src/main/res/values-nb/strings-nb.xml]: Done

Comment (ahaarrestad) [reply app/src/main/res/values-nb/strings-nb.xml]: Done

Comment (ahaarrestad) [reply app/src/main/res/values-nb/strings-nb.xml]: Line 680. Done

Comment (ahaarrestad) [reply app/src/main/res/values-nb/strings-nb.xml]: Line 688. Done

Comment (ahaarrestad) [reply app/src/main/res/values-nb/strings-nb.xml]: Line 690. Done

Comment (ahaarrestad) [reply app/src/main/res/values-nb/strings-nb.xml]: Line 693. Done

Comment (ahaarrestad) [reply app/src/main/res/values-nb/strings-nb.xml]: Line 694. Done

Comment (ahaarrestad) [reply app/src/main/res/values-nb/strings-nb.xml]: Line 704. Done

Comment (ahaarrestad) [reply app/src/main/res/values-nb/strings-nb.xml]: Done

**Comment by ahaarrestad** [comment]
Thank's for your review, @dabear it is hard to get all the translations right, and this review helped a lot :-)

**Comment by ahaarrestad** [comment]
The build timed out in the pullrequest yesterday, but was OK in my fork, @jamorham . Added new translations today and hope the build is passing here as well. Happy with the changed, @dabear ?

**Comment by ahaarrestad** [comment]
The build has passed now, @jamorham .

File: app/src/main/res/values-nb/strings-nb.xml [hunk 18]
```diff
@@ -1,501 +1,848 @@
...
+
+	<!-- wifi Settings -->
+	<string name="wifi_recievers_dialog_message">Kommaseparert liste av ip:port (for eksempel 37.142.132.220:50005,37.142.132.220:50010,mongodb://user:pass@ds053958.mongolab.com:53958/db/collection)</string>
 	<string name="wifi_recievers_dialog_title">Skriv inn IP-adresser og porter mottakere (inkludert mongodb adresser hvis nødvendig)</string>
-	<string name="location_not_found_title">Sted er ikke aktivert</string>
-	<string name="location_not_found_message">For Bluetooth oppdagelse å jobbe med nyere enheter, må plasseringen være aktivert. xDrip ikke spore posisjonen din, og det kan deaktiveres etter paringen er vellykket.</string>
+
+	<!-- Location Settings -->
+	<string name="location_not_found_title">Stedstjenester er ikke aktivert</string>
+	<string name="location_not_found_message">For at blåtannoppdagelse skal virke med nyere enheter, må stedstjenester være aktivert. xDrip kan ikke spore posisjonen din, og det kan deaktiveres etter at paringen er vellykket.</string>
```
Comment (dabear) [inline app/src/main/res/values-nb/strings-nb.xml]: blåtannsøk?

File: app/src/main/res/values-nb/strings-nb.xml [hunk 19]
```diff
@@ -1,501 +1,848 @@
...
+	<string name="sync_using_custom_security_key">Synkroniser med egendefinert sikkerhetsnøkkel</string>
+	<string name="handset_group_security_key">Gruppesikkerhets syknroniseringsnøkkel for håndsett</string>
+	<string name="handset_sync_grouping_key">Syknroniserings gruppenøkkel for håndsett</string>
+	<string name="alert_name_colon">Alarmnavn:</string>
 	<string name="threshold_colon">Terskel:</string>
-	<string name="default_snooze_colon">Standard Snooze:</string>
-	<string name="re_raise_every_x_minutes_if_unaknowledged">Re-raise hver\nx minutter hvis\nunaknowledged:</string>
-	<string name="alert_tone_colon">Alert Tone:</string>
+	<string name="default_snooze_colon">Standard Utsettelse:</string>
+	<string name="re_raise_every_x_minutes_if_unaknowledged">Aktiver hvert \nx-te minutter hvis \nikke b
```
Comment (dabear) [inline app/src/main/res/values-nb/strings-nb.xml]: blåta**_nn_**

Comment (dabear) [inline app/src/main/res/values-nb/strings-nb.xml]: is cddata correct here??

Comment (dabear) [inline app/src/main/res/values-nb/strings-nb.xml]: insulin

File: app/src/main/res/values-nb/strings-nb.xml [hunk 20]
```diff
@@ -1,501 +1,848 @@
...
 	<string name="view_events_log">Se Aktivitetslogg</string>
-	<string name="share_config_via_qr_code">Del config via QR-kode</string>
+	<string name="share_config_via_qr_code">Del konfigurasjon med QR-kode</string>
 	<string name="check_for_updated_version">Sjekk om det finnes oppdatert versjon</string>
 	<string name="send_feedback_to_developer">Send tilbakemelding til utvikleren</string>
 	<string name="home_screen">Startskjermen</string>
-	<string name="calibration_graph">Kalibrering Graph</string>
-	<string name="glucose_units">Glukose Units</string>
+	<string name="calibration_graph">Kalibreringsgraf</string>
+	<string name="glucose_units">Glukose Enheter</string>
```
Comment (dabear) [inline app/src/main/res/values-nb/strings-nb.xml]: glukoseenheter

File: app/src/main/res/values-nb/strings-nb.xml [hunk 21]
```diff
@@ -1,501 +1,848 @@
...
 	<string name="xdrip_plus_sync_settings">xDrip+ Synkroniseringsinnstillinger</string>
 	<string name="xdrip_plus_update_settings">xDrip+ Oppdater innstillinger</string>
 	<string name="automatic_update_check">Automatisk oppdatering sjekk</string>
-	<string name="update_channel">Oppdater Channel</string>
-	<string name="send_crash_errors_to_developer">Send krasjfeil til utvikler</string>
-	<string name="automatic_crash_reporting">Automatic Crash Reporting</string>
-	<string name="send_feedback_message">Send tilbakemelding Message</string>
+	<string name="update_channel">Oppdateringskanal</string>
+	<string name="send_crash_errors_to_developer">Send kræsjfeil til utvikler</string>
+	<string name="automatic_crash_reporting">Automatisk kræsjrapportering</string>
```
Comment (dabear) [inline app/src/main/res/values-nb/strings-nb.xml]: krasj is the correct spelling

File: app/src/main/res/values-nb/strings-nb.xml [hunk 22]
```diff
@@ -1,501 +1,848 @@
...
-	<string name="load_save_settings_to_sdcard">Belastning / lagre innstillinger til SDcard</string>
-	<string name="alarms_and_alerts">Alarmer og varslinger</string>
-	<string name="alerts_and_notifications">Varsler og meldinger</string>
-	<string name="glucose_level_alerts_list">Glukosenivå varsler Liste</string>
-	<string name="glucose_alerts_settings">Glukose Varsler Innstillinger</string>
-	<string name="smart_snoozing">Smart snoozing</string>
-	<string name="calibration_alerts">Kalibrerings varsler</string>
-	<string name="missed_reading_alert">Tapte leser Alert</string>
+	<string name="show_settings_qr_codes">Vis Innstillings QR-koder</string>
+	<string name="load_save_settings_to_sdcard">Last / lagre innstillinger til SDcard</string>
```
Comment (dabear) [inline app/src/main/res/values-nb/strings-nb.xml]: last inn

File: app/src/main/res/values-nb/strings-nb.xml (hunk 18 above)
Comment (ahaarrestad) [reply app/src/main/res/values-nb/strings-nb.xml]: Done

File: app/src/main/res/values-nb/strings-nb.xml (hunk 19 above)
Comment (ahaarrestad) [reply app/src/main/res/values-nb/strings-nb.xml]: Line 584: Done

Comment (ahaarrestad) [reply app/src/main/res/values-nb/strings-nb.xml]: Line 760: 
English: <![CDATA[Notify when data a
Norwegian: <![CDATA[Gi beskjed når 

I believe this is correct. or is it another line?

Comment (ahaarrestad) [reply app/src/main/res/values-nb/strings-nb.xml]: Line 786: Done

File: app/src/main/res/values-nb/strings-nb.xml (hunk 20 above)
Comment (ahaarrestad) [reply app/src/main/res/values-nb/strings-nb.xml]: Done

File: app/src/main/res/values-nb/strings-nb.xml (hunk 21 above)
Comment (ahaarrestad) [reply app/src/main/res/values-nb/strings-nb.xml]: Done

File: app/src/main/res/values-nb/strings-nb.xml (hunk 22 above)
Comment (ahaarrestad) [reply app/src/main/res/values-nb/strings-nb.xml]: Done

**Comment by jamorham** [comment]
Thanks for your continued work on this guys, I know translations are fiddly and tough to get right.

**Comment by ahaarrestad** [comment]
@jamorham the pull request builds in my fork. You can either check https://travis-ci.org/ahaarrestad/xDrip-plus/builds/327398378 or try a retrograde of this build. Should be OK to merge the pull request now.

**Comment by dabear** [comment]
Looking good

**Pull request merged**